tests/unit/button/button_core.js

```diff
@@ -6,50 +6,54 @@
 
 module("button: core");
 
-test("checkbox", function() {
+test("checkbox", function( assert ) {
 	expect( 4 );
 	var input = $("#check"),
 		label = $("label[for=check]");
 	ok( input.is(":visible") );
 	ok( label.is(":not(.ui-button)") );
 	input.button();
-	ok( input.is(".ui-helper-hidden-accessible") );
-	ok( label.is(".ui-button") );
+	assert.hasClasses( input, "ui-helper-hidden-accessible" );
+	assert.hasClasses( label, "ui-button" );
 });
 
-test("radios", function() {
-	expect( 4 );
+test("radios", function( assert ) {
+	expect( 8 );
 	var inputs = $("#radio0 input"),
 		labels = $("#radio0 label");
 	ok( inputs.is(":visible") );
 	ok( labels.is(":not(.ui-button)") );
 	inputs.button();
-	ok( inputs.is(".ui-helper-hidden-accessible") );
-	ok( labels.is(".ui-button") );
+	inputs.each(function() {
+		assert.hasClasses( this, "ui-helper-hidden-accessible" );
+	});
+	labels.each(function() {
+		assert.hasClasses( this, "ui-button" );
+	});
 });
 
-function assert(noForm, form1, form2) {
-	ok( $("#radio0 .ui-button" + noForm).is(".ui-state-active") );
-	ok( $("#radio1 .ui-button" + form1).is(".ui-state-active") );
-	ok( $("#radio2 .ui-button" + form2).is(".ui-state-active") );
+function assertClasses(noForm, form1, form2) {
```

Comment:
You shouldn't use `QUnit.assert` to execute assertions (details elsewhere/some other time). In this case I suggest moving this helper into the one test that uses it, so that you have the `assert` argument accessible from the parent scope.